README.md

```diff
@@ -305,6 +293,51 @@ will return:
 ```
 
 
+#### Invite a member
+A member can be invited:
+
+```
+POST /circles/:circleId/members
+Authorization: Basic `base64(id:token)`
+{
+    userId: {userId}
+}
+```
+
+this will result in:
+
+```
+204 NO CONTENT
+```
+
+### Accept invitation
+A member can accept a pending invitation. Only the member who was invited can accept.
+
+```
+POST /circles/:circleId/members/accept
+Authorization: Basic `base64(id:token)`
+```
+
+this will result in:
+
+```
+204 NO CONTENT
+```
+
+### Remove member
+A member can be removed by:
+
+```
+DELETE /circles/:circleId/members/:userId
+Authorization: Basic `base64(id:token)`
+```
+
+this will result in:
+
+```
+204 NO CONTENT
+```
```

Comment:
Does this also remove members who are in the `invited` list? (I hope yes. But it should be made explicit in the text.)